### PR TITLE
Add support for multiple selection picker

### PR DESCRIPTION
**Pull request by chirag04**
@oblador

### CODE REVIEW

File: CJActionSheetPicker/CJActionSheetPicker/CJActionSheetPicker.m [hunk 1]
```diff
@@ -33,28 +34,56 @@ - (dispatch_queue_t)methodQueue
...
+
+        void(^doneBlock)(ActionSheetStringMultipleSelectionPicker *, NSArray*, id) = ^(ActionSheetStringMultipleSelectionPicker *picker, NSArray *selectedIndices, id selectedValues) {
+            resolve(@{
+                @"cancelled": @NO,
+                @"selectedIndices": selectedIndices,
+                @"selectedValues": (NSArray *)selectedValues
+            });
+        };
+
+        [ActionSheetStringMultipleSelectionPicker showPickerWithTitle:title
```
Comment (chirag04) [inline CJActionSheetPicker/CJActionSheetPicker/CJActionSheetPicker.m:59]: this can be outside the if. just set the correct `initialSelection` in if.

Comment (oblador) [reply CJActionSheetPicker/CJActionSheetPicker/CJActionSheetPicker.m:59]: Not really? There's different classes for single and multiple selections.

Comment (chirag04) [reply CJActionSheetPicker/CJActionSheetPicker/CJActionSheetPicker.m:59]: oh damn. I need to clean my glasses. haha

**Comment by chirag04** [comment]
PR welcome for documentation :D

**Pull request merged**